engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -142,7 +146,7 @@ namespace dmGameSystem
         // Used after resolving info from all textures
         dmhash_t                        m_AnimationID;                      // The animation of the driving atlas
         uint32_t                        m_Frames[MAX_TEXTURE_COUNT];        // The resolved frame indices
-        uint32_t                        m_PageIndices[MAX_TEXTURE_COUNT];
+        float                           m_PageIndices[MAX_TEXTURE_COUNT];
```

Comment:
The attribute writer function only takes floats as input (mostly for simplicity), so I changed the type here so we can pass the pointer in directly, similar to texture coordinates.